### ISSUE TITLE
I can't install System Patches offline

### ISSUE THREAD

**Issue by APXD6** [exelix11/SwitchThemeInjector#125]
I followed [this](https://github.com/exelix11/theme-patches) tutorial but it isn't working, the app doesn't detect the folder. English isn't my first language so perhaps I missed something. Sorry to bother.

**Comment by exelix11** [comment]
What firmware ? Do you have latest theme installer ?
Do you get any specific error message ?

**Comment by APXD6** [comment]
With firmware do you refer to the "console version", right? It is on 15.0.1, the latest.
I do have the latest theme installer and no, there's no error. Actually, there's no reaction at all, the folder appears in the themes section but it does not detect it as system patches.

> What firmware ? Do you have latest theme installer ? Do you get any specific error message ?

**Comment by exelix11** [comment]
If you get no error it means the installation was successful, try installing a theme and see what happens

**Comment by APXD6** [comment]
It didn't work
![20221207_191747](https://user-images.githubusercontent.com/119984417/206309223-101782b9-6ac7-4ac0-a90c-676f3bcc1649.jpg)
In the app menu it said that it was on version 2.6.4, so I don't think it got updated.
I'm trying to use [this theme](https://themezer.net/packs/P5G-Persona-5-Golden-version-2-4f7) btw, just in case it matters.

**Comment by exelix11** [comment]
Old theme installer doesn't support this way of distributing patches you must use version 2.7 which is the latest release.

**Comment by APXD6** [comment]
You're right, my bad. I thought I got the latest version.
I'm going to test if everything works now.

**Comment by APXD6** [comment]
I tried everything and it still didn't work. I'm now using the previous version of the [theme](https://themezer.net/packs/P5G-Persona-5-Golden-4e9) because the newer one used custom icons. I made sure to delete the right folders, to uninstall everything, to reboot after unistalling and to extract home menu and it is still not working.
The error code it's the same as before, don't know if that is important.
